ofrak_components/ofrak_components/ecc/reedsolomon.py

```diff
@@ -33,6 +33,3 @@ def decode(self, payload: bytes) -> bytes:
             return self.RSC.decode(data=payload)[0]
         except rs.ReedSolomonError:
             raise EccError()
-
-    def check(self, payload: bytes) -> bool:
-        return self.RSC.check(data=payload)
```

Comment:
@jaredgonzales how important is this code? 

Even though there don't appear to be any uses of it in tests, should it be expected to remain as part of the API? If so, we should add tests for it instead of removing it.

Reply:
It can be removed since it's not used anywhere. That part was added before the flash component fully implemented ecc support.